editor/register_editor_types.cpp

```diff
@@ -212,6 +212,9 @@ void register_editor_types() {
 	EditorPlugins::add_by_type<Skeleton2DEditorPlugin>();
 	EditorPlugins::add_by_type<Sprite2DEditorPlugin>();
 	EditorPlugins::add_by_type<TilesEditorPlugin>();
+
+	// For correct doc generation.
```

Comment:
If it is the correct place, please move these definitions here too.

https://github.com/godotengine/godot/blob/c1128e911ccd6f1e8c35646df804d894652a58f1/core/config/project_settings.cpp#L1317-L1319

Reply:
Will do, see my reasoning above